includes/class-sensei-posttypes.php

```diff
@@ -688,7 +688,7 @@ public function setup_course_category_taxonomy() {
 			'show_in_nav_menus' => true,
 			'capabilities'      => array(
 				'manage_terms' => 'manage_categories',
-				'edit_terms'   => 'edit_courses',
+				'edit_terms'   => 'manage_course_categories',
```

Comment:
Compared to the modules approach, I needed to use a different approach here, otherwise, in the editor sidebar the button "Add New Course Category" would be hidden.